### PR TITLE
chore: Switch to struct args from builder

### PR DESCRIPTION
**Pull request by pimeys**
# Description

Makes it simpler to use the command line arguments.

# Type of change

- [ ] 💔 Breaking
- [ ] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [x] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

File: cli/crates/cli/src/cli_input.rs [hunk 1]
```diff
@@ -1,75 +1,193 @@
...
+    }
+
+    fn completions_for_shell(generator: impl Generator) {
+        clap_complete::generate(generator, &mut Args::command(), "grafbase", &mut std::io::stdout());
+    }
+}
+
+#[derive(Debug, Parser)]
+pub struct CompletionsCommand {
+    /// The shell to generate completions for.
```
Comment (yoav-lavi) [inline cli/crates/cli/src/cli_input.rs:55]: If we're not adding the second line here I think we can remove the dot

**Review by yoav-lavi — APPROVED** [review]
Really nice work!
Could we diff the help output between old and new to make sure it acts the same?
Possibly would be good to have a snapshot for it

**Comment by pimeys** [comment]
# Screenshots from this branch

**Comment by pimeys** [comment]
# Screenshots from this branch

**Pull request merged**